test/index.js

```diff
@@ -63,6 +71,32 @@ describe('Klaviyo', function () {
         .end(done);
     });
 
+    it('should send the Placed Order event correctly for Completed Order event', function(done){
+      var json = test.fixture('track-completed-order');
+
+      test
+        .set(settings)
+        .track(json.input)
+        .request(0)
+        .query('data', json.output.order, decode)
+        .expects(200)
+        .end(done);
+
+    });
+
+    it('should send the Ordered Product event correctly for Completed Order event', function(done){
```

Comment:
This test doesn't ensure that multiple Ordered Product events are sent, add another test that does